src/CopyableInput/CopyableInput.tsx

```diff
@@ -25,6 +25,7 @@ const propTypes = {
   enableCopy: PropTypes.bool,
   required: PropTypes.bool,
   size: PropTypes.string,
+  onClickCopy: PropTypes.func,
```

Comment:
We should add a definition to the Props interface as well for Typescript support!

Reply:
Ah, now I see the interface [above](https://github.com/Clever/components/pull/442/files#diff-88afd73593c0110faa0f9eb8e3632e64R12).

Reply:
Ah good call, thanks 👍
12225f3